lib/nuts.js

```diff
@@ -241,13 +241,22 @@ Nuts.prototype.onUpdateRedirect = function(req, res, next) {
     .fail(next);
 };
 
+function determineChannel(req) {
+    console.log("determineChannel", req);
+    if(req.params.version && req.params.version.indexOf('beta') === -1) {
```

Comment:
```suggestion
    if(req.params.version && req.params.version.indexOf('beta') !== -1) {
```

Reply:
Sure, if you want it to work correctly. :-)